[doc] Note that `coq_makefile --help` will display help. (#14067)
coq/coq: doc/sphinx/practical-tools/utilities.rst

example, this allows to pass the two-word option ``-w all`` (see
:ref:`command line options <command-line-options>`).

> Note that, in general, it is mandatory to specify a ``-R/-Q`` flag for
> your project. Omitting it will generate files that are not in general
> usable unless you are developing Coq itself.

Reviewer: ```suggestion
Generally, you should specify a ``-R/-Q`` flag for
your project. Omitting it generates files that are not generally
usable unless you are developing Coq itself.
```

"in general" and "it is mandatory" are contradictory.

> Note that, in general, it is mandatory to specify a ``-R/-Q`` flag for
> your project, failing to do will produce files without a proper
> library prefix, which are to be used only in special cases.

Reviewer: ```suggestion
Normally you should specify a ``-R/-Q`` flag for
your project.  Omitting it will generate compiled files without a proper
library prefix, which is for use in special cases.
```

Who would use the "special cases" and why?

Author: Should is not OK, it is indeed mandatory. Special cases means "I am a developer of Coq and know what I am doing"

Author: Actually seems like a good idea to make `coqc foo.v` fail if there is no scope, but the compat impact is huge here.

  the contents of the ``_CoqProject`` file as well as the path relevant to
  Coq.

> See ``coq_makefile --help`` for a description of command line options.

Reviewer: ```suggestion
Run ``coq_makefile --help`` for a description of command line options.
```

> See ``coq_makefile --help`` for help on particular command line options.

Reviewer: ```suggestion
See ``coq_makefile --help`` for a description of command line options.
```
